netbout-web/pom.xml

```diff
@@ -733,6 +733,7 @@
                         <artifactId>qulice-maven-plugin</artifactId>
                         <configuration>
                             <excludes>
+                                <exclude>checkstyle:/src/main/java/com/netbout/rest/TkStart.java</exclude>
```

Comment:
@prahladyeri did you try to remove the exclusion as @carlosmiranda [suggested](https://github.com/yegor256/netbout/pull/921#discussion_r49829400) ?